### ISSUE TITLE
Cannot search for a proposed relation

### ISSUE THREAD

**Issue by tomas-knap** [odalic/sti#456]
When you propose new class, you cannot search it: 

Steps:
1) Run for books-v1
2) Propose new class e.g. Person123 in dbpedia Jan. Try to search for that

![screen shot 2017-05-30 at 10 33 08](https://cloud.githubusercontent.com/assets/3014917/26574936/964498cc-4523-11e7-8c96-1ab1af321059.png)

@JanVana I thought this was solved? Any comments to that?
At least pls let me know why that does not work

**Comment by tomas-knap** [comment]
Probably caused by https://github.com/odalic/sti/issues/459

**Comment by JanVana** [comment]
A cannot reproduce this issue on my side. I can find the Person123 and also any newly proposed resource/class. I have also verified that the proposed resources are saved. If anyone else have the same problem, please comment here.

**Comment by JanVana** [comment]
Person 123 properties:
```
select ?property ?value {<http://odalic.eu/schema/4cc9106a-8bfc-44c7-bab6-3066db53dc7d> ?property ?value}
```

**Comment by brodecva** [comment]
Hi, well I think the issue might be the fact that an exact match must be invoked. For example if I had added class with two-word label. I could not find it using neither the first nor the second word, but could if yousing the whole label as the search term.
I think this is a natural limitation we have accepted, but it might be wise to check if it is properly documented, @JanVana .

**Comment by tomas-knap** [comment]
Yes, that is what I know about. 

Works for classes/disambiguated entities, but not for relations. 

Try to add new relation and then search for it. 

Probably connected with wrong storing of relations: See: https://github.com/odalic/sti/issues/459

**Comment by brodecva** [comment]
Well I tried even the properties, and it worked for me look it up later (using the exact search of course).

**Comment by JanVana** [comment]
I suspect that this issue will be resolved once the issue #459 is resolved.
